src/sentry/features/temporary.py

```diff
@@ -76,6 +76,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:codecov-commit-sha-from-git-blame", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
     # Enable continuous profiling
     manager.add("organizations:continuous-profiling", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+# Enable continuous profiling
+    manager.add("organizations:continuous-profiling-allowed-ingest", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
```

Comment:
Is this supposed to be flagpole or internal?

Reply:
internal, just a leftover. I'll fix it

Reply:
Why an internal feature flag? Flagpole will allow us to enable it based on a list of orgs without any additional boiler plate that an internal feature flag requires.

Reply:
Because as far as I understood it should be enabled/disabled automatically for a certain date range.